.eslintrc

```diff
@@ -0,0 +1,40 @@
+{
+  "extends": "airbnb/legacy",
+  "env": {
+    "es6": true,
+    "node": true,
+    "browser": true
+  },
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "ecmaFeatures": {
+      "experimentalObjectRestSpread": true
+    }
+  },
+  "rules": {
```

Comment:
In general, I don't like veering from the AirBnB rules too much. Please look at the rules we use in account portal and other ecosystem projects.